### ISSUE TITLE
QA DocAI can't work because of NaN's (and little update request)

### ISSUE THREAD

**Issue by skiiiks** [GoogleCloudPlatform/generative-ai#217]
In this example:
https://github.com/GoogleCloudPlatform/generative-ai/blob/main/language/use-cases/document-qa/question_answering_documentai_vector_store_palm.ipynb

1) You have to update the import of `google-cloud-documentai-toolbox` from `0.10.1` of the document to `0.10.3a`, if not, it doesn't go ;-)

2) It crashes at one point:

```py
pdf_data_sample["embedding"] = pdf_data_sample["chunks"].apply(
    lambda x: embedding_model_with_backoff([x])
)
```

Investigating, it comes from the chunks, there are "NaNs" in the chunks ... and there are because when handling the PDF, it is only able to grab the first page, the rest are all "null" values.

Any solution / workaround ?

**Comment by holtskinner** [comment]
I already fixed issue 1 in #226

And thanks for the context on issue 2. I'll see if I can resolve the issue.

**Comment by SampathkumarSubramaniam** [comment]
> I already fixed issue 1 in #226
> 
> And thanks for the context on issue 2. I'll see if I can resolve the issue.
Any news on issue 2. I could reproduce from my side following https://cloud.google.com/blog/products/ai-machine-learning/ask-your-documents-document-ai-and-palm2-for-question-answering
